.github/workflows/php-downstream.yml

```
@@ -5,10 +5,10 @@ on:
 
 jobs:
   build:
-    strategy:      
+    strategy:
       matrix:
         php-versions: ["7.2", "7.3", "7.4", "8.0"]
-        symfony-versions: ["~3.0", "~4.0", "~5.0"]
+        symfony-versions: ["~3.0", "~4.0", "~5.0", "~6.0"]
```

Ugh. Symfony 6 requires PHP >= 8.0, so we'll need to add some excludes to the build configuration.

```suggestion
        symfony-versions: ["~3.0", "~4.0", "~5.0", "~6.0"]
        exclude:
          - php-versions: "7.2"
            symfony-versions: "~6.0"
          - php-versions: "7.3"
            symfony-versions: "~6.0"
          - php-versions: "7.4"
            symfony-versions: "~6.0"
```